src/runtime/utils/generateTypes.ts

```diff
@@ -0,0 +1,26 @@
+import { genInterface } from 'knitwork'
+import { createResolver } from '@nuxt/kit'
+import type { AuthProviders } from '../types'
+
+export function generateModuleTypes(provider: AuthProviders) {
+  const { resolve } = createResolver(import.meta.url)
+
+  const providerSpecificTypes: string[] = []
+
+  if (provider.type === 'authjs') {
+    providerSpecificTypes.push(`  const { getServerSession, getToken, NuxtAuthHandler }: typeof import('${resolve('./runtime/server/services/authjs')}')`)
```

Comment:
Question - does this resolve relative to current path or from root? From the docs, it looks like the resolution will happen from the current path which is not correct:
https://nuxt.com/docs/api/advanced/import-meta#builder-properties